run

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
```

Comment:
Hmm is this a duplicate of the run.sh script ? We should be able to remove this with git rm run